modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+[id="nw-ingress-controller-configuration-parameters_{context}"]
+= Ingress controller configuration parameters
+
+The `ingresscontrollers.operator.openshift.io` resource offers the following
+configuration parameters.
+
+[cols="3a,8a",options="header"]
+|===
+|Parameter |Description
+
+|`domain`
+|`domain` is a DNS name serviced by the Ingress controller and is used to configure multiple features:
+
+* For the `LoadBalancerService` endpoint publishing strategy, domain is used to configure DNS records. See `endpointPublishingStrategy`.
+
+* When using a generated default certificate, the certificate will be valid for domain and its subdomains. See `defaultCertificate`.
+
+* The value is published to individual Route statuses so that end-users know where to target external DNS records.
+
+`domain` must be unique among all Ingress controllers, and cannot be updated.
+
+If empty, defaults to `ingress.config.openshift.io/cluster` `.spec.domain`.
+
+This parameter is optional.
+
+|`replicas`
+|`replicas` is the desired number of Ingress controller replicas. If not set, defaults to 2.
+
+|`endpointPublishingStrategy`
+|`endpointPublishingStrategy` is used to publish the Ingress controller endpoints to other networks, enable load balancer integrations, and so on.
+
+If not set, the default is based on `infrastructure.config.openshift.io/cluster` `.status.platform`:
+
+* AWS: `LoadBalancerService` (with External scope)
+* Azure: `LoadBalancerService` (with External scope)
+* GCP: `LoadBalancerService` (with External scope)
+* Any other platform types (including None) default to `HostNetwork`.
+
+`endpointPublishingStrategy` cannot be updated.
+
+This parameter is optional.
+
+|`defaultCertificate`
+|`defaultCertificate` is a reference to a secret containing the default certificate served by the Ingress controller. When Routes do not specify their own certificate, `defaultCertificate` is used.
```

Comment:
s/containing/that contains
s/served/that is served
s/`defaultCertificate`/the `defaultCertificate` value